test/dist_utils.py

```diff
@@ -51,25 +50,29 @@ def set_termination_signal():
     _TERMINATION_SIGNAL.set()
 
 
-def dist_init(test_method=None, setup_model_parallel=True, clean_shutdown=True):
+def dist_init(old_test_method=None, setup_model_parallel=True, clean_shutdown=True):
```

Comment:
Why the rename?

Reply:
Not fully intended, I tink I picked #27940 instead of #27940 when resolving conflicts.

Reply:
I see. I don't have a preference, but don't like anything with `old` or `new` in it, because it implies something is either being introduced or being deprecated. And for these decorators it's not needed if you use the functools wrappers.